libs/sdk-ui-vis-commons/src/legend/ColorLegend.tsx

```diff
@@ -65,11 +66,37 @@ export function ColorBoxes(colorBoxProps: IColorBoxesProps): JSX.Element {
     );
 }
 
+function renderLegendBoxes(
```

Comment:
Suggestion: This is effectively another component, right? Might be worth it calling and treating it like a component.

Reply:
fixed with ae2b045

Reply:
Fixed